ui/page/send/page.go

```diff
@@ -52,9 +52,11 @@ type Page struct {
 
 	moreOptionIsOpen bool
 
-	exchangeRate   float64
-	usdExchangeSet bool
-	exchangeError  string
+	exchangeRate          float64
+	usdExchangeSet        bool
+	exchangeError         string
+	defaultAcccountNumber int32
+	currentAccountNumber  int32
```

Comment:
Rather than have this here, you can simple add an `OnChanged()` to the account selector modal, and returns a bool when ever the account changes.